src/Core/OrganizationFeatures/OrganizationSubscriptionUpdate/Interface/IUpdateSecretsManagerSubscriptionCommand.cs

```diff
@@ -5,4 +5,5 @@ namespace Bit.Core.OrganizationFeatures.OrganizationSubscriptionUpdate.Interface
 public interface IUpdateSecretsManagerSubscriptionCommand
 {
     Task UpdateSecretsManagerSubscription(SecretsManagerSubscriptionUpdate update);
+    Task AdjustSecretsManagerServiceAccountsAsync(Guid organizationId, int smServiceAccountsAdjustment);
```

Comment:
We probably don't need to repeat `SecretsManager` in these method names. How about
* `UpdateSubscriptionAsync`
* `AdjustSeatsAsync`